src/Voice/AudioEncoder.js

```diff
@@ -54,26 +55,33 @@ export default class AudioEncoder {
 				return choice;
 			}
 		}
+		
+		console.error("avconv or ffmpeg not found, please install them to use audio!");
```

Comment:
This isn't necessary, it could be handled by the calling method when this method.